coderd/database/lock.go

```diff
@@ -10,6 +10,7 @@ const (
 	LockIDEnterpriseDeploymentSetup
 	LockIDDBRollup
 	LockIDDBPurge
+	LockIDReportGenerator
```

Comment:
I think we should name this more specifically.

```suggestion
	LockIDNotificationsReportGenerator
```